README.md

```diff
@@ -4,56 +4,57 @@
 
 ## What is a short name
 
-When tools like Podman or Docker pull container images, users prefer to use
+When tools like [Podman][podman-gh] or [Docker][[docker-cli-gh]] pull container images, users prefer to use
 short names like `fedora` or `alpine` rather then fully specified image names
-registry.fedoraproject.org/fedora and docker.io/alpine respectively. In
+`registry.fedoraproject.org/fedora` and `docker.io/alpine`, respectively. In
 container engines that allow you to specify more then a single registry for
-storing container images, using short names can lead to ambiguity. Imageine
-that I have two registries defined and both container an image names foobar.
+storing container images, using short names can lead to ambiguity.
 
-Now if I specify foobar, I am not sure which image I will get. There is potential for malicious parties to take advantage of this by spoofing images and tricking users
-into pulling them.
+Imagine that I have two registries defined and both contain an image named
+"foobar". Now if I specify `foobar`, I am not sure which image I will
+get. There is potential for malicious parties to take advantage of this by
+spoofing images and tricking users into pulling them.
 
-We are building into these contaienr engine tools the ability to use short name
+We are building into these container engine tools the ability to use short name
 aliases to help mitigate the risk of pulling the wrong image, especially when
 the image is a well known short name.
 
-Similar to aliases in BASH, a short-name alias has a left-hand name that is
+Similar to aliases in shells such as [Bash][gnu-bash], a short-name alias has a left-hand name that is
 being replaced with a right-hand value. Short-name aliases can now be
-configured in the registries.conf file as follows:
+configured in the 'registries.conf' file as follows:
 
 ```
-unqualified-search-registries=[“registry.fedoraproject.org”, “docker.io”]
+unqualified-search-registries=['registry.fedoraproject.org', 'docker.io']
 
 [aliases]
-“fedora”=”registry.fedoraproject.org/fedora”
+'fedora'='registry.fedoraproject.org/fedora'
```

Comment:
I prefer double quotes `"` in toml, but that's a non blocking nit.

Reply:
I updated it to use double quotes in my latest rebase.